AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -20,6 +21,10 @@ protocol MeasurementStreamStorageContextUpdate {
     func updateSessionEndtime(_ endTime: Date, for sessionUUID: SessionUUID) throws
     func updateSessionFollowing(_ sessionStatus: SessionFollowing, for sessionUUID: SessionUUID)
     func existingMeasurementStream(_ sessionUUID: SessionUUID, name: String) throws -> MeasurementStreamLocalID?
+    func preapareSessionForDeletion(_ sessionUUID: SessionUUID) throws
```

Comment:
it suggests some database delete action, a bit misleading. Maybe `markSessionForDelete`?